build.cake

```diff
@@ -48,6 +48,7 @@ class ExtensionSpec
     public List<string> Categories { get; set; }
     public string AnalyzedPackageVersion { get; set; }
     public string TargetCakeVersion { get; set; }
+    public List<string> TargetFrameworkVersions { get; set; }
```

Comment:
Today I learned that TFM stands for "Target Framework Moniker" ([here](https://docs.microsoft.com/en-us/dotnet/standard/frameworks)). So how about we name this property `TargetFrameworkMonikers`?

Reply:
`TargetFrameworkMonikers` works for me, as well as `TargetFrameworks` which is what holds the TFMs in the build.

```
  <PropertyGroup>
    <TargetFrameworks>net5.0;netstandard2.0;net461</TargetFrameworks>
  </PropertyGroup>
```

Reply:
`TargetFrameworks` is shorter, I like it.

Reply:
Works both for me. How about going with `TargetFrameworks`?

Reply:
I vote for `TargetFrameworks`. It's shorter, it's similar to the XML node in.csproj files. It's better than my suggestion.

Reply:
Updated to `TargetFrameworks`